impl/src/planning.rs

```diff
@@ -212,8 +212,8 @@ impl CrateCatalogEntry {
   pub fn local_build_path(&self, settings: &RazeSettings) -> String {
     match settings.genmode {
       GenMode::Remote => format!(
-        "remote/{}.{}",
-        &self.package_ident, settings.output_buildfile_suffix,
+        "remote/BUILD.{}.bazel",
```

Comment:
Why not `{}.BUILD.bazel`? And then why simply not update the default configuration for output_buildfile_suffix from `BUILD` to `BUILD.bazel`?

Reply:
I like the idea that all build files start with "BUILD" I think it improves the discoverability of them. Also, wasn't "output_buildfile_suffix" updated in #205?

Reply:
I have missed #205 
I feel othewise, the reading `openssl.BUILD.bazel` feels more natural than `BUILD.opensssl.bazel`, in addition finding all the build file would be `*BUILD.bazel`.

Let try to bring in more opinion from the rules_rust folks: @acmcarther @dfreese @smklein @mfarrugi

Reply:
Becasue bazel didn't enforce the `.bazel` suffix I think the thing users end up searching would be `BUILD*` (that's what I do for `find`, `grep`, and generally searching for the correct `BUILD` file in Vscode. I can change it if you feel strongly about it but I'd prefer the current implementation. Let me know and/or I'll wait for a response from one of the people you pinged.

Reply:
Since we have to different point of view on this and it is in the end of question of point of view, let's wait and gather more opinion, and see which way to go.

Reply:
@acmcarther @dfreese @smklein @mfarrugi anyone available to chime in?